infra/lib/prow-service-accounts.ts

```diff
@@ -50,6 +50,34 @@ export class ProwServiceAccounts extends cdk.Construct {
       ],
     });
 
+    const postEcrPublicPolicy = new iam.PolicyStatement({
+      actions: [
+        // Read access
+        "ecr-public:Describe*",
+        "ecr-public:Get*",
+        // Limited write access
+        "ecr-public:PutRepositoryCatalogData",
+        "ecr-public:UploadLayerPart",
+        "ecr-public:CompleteLayerUpload",
+        "ecr-public:InitiateLayerUpload",
+        "ecr-public:PutImage",
+        "ecr-public:ListTagsForResource",
+        "ecr-public:PutRegistryCatalogData",
+        "ecr-public:BatchCheckLayerAvailability"
+      ],
+      resources: [
+        `arn:${props.stackPartition}:ecr-public::${process.env.CDK_DEFAULT_ACCOUNT}:registry/*`,
```

Comment:
I've read that this environment variable does not always exist. Elsewhere in the CDK I pass down the `account` as part of props. https://github.com/aws-controllers-k8s/test-infra/blob/8c107c357e86233ac4ccf1a8a56d34eaa793dc69/infra/lib/test-ci-stack.ts#L20

Reply:
Done.